Kumi.Game/Screens/Select/ChartSetInfoWedge.cs

```diff
@@ -15,10 +15,10 @@ public partial class ChartSetInfoWedge : Container
 
     public ChartSetInfoWedge()
     {
-        X = -5;
+        X = 5;
         Masking = true;
         CornerRadius = 5;
-        Margin = new MarginPadding { Left = -5 };
```

Comment:
i get the difference, but why the change? is it really a fix if it's kinda just done out of thin air?